### PR TITLE
Issue22/add lic 15

### PR DESCRIPTION
**Pull request by angelicaengstrom**
My proposal for issue #22 regarding LIC 15, this includes

* functionality for LIC 15

* javadoc

* false and positive unit-tests

Please review and suggest feedback

### CODE REVIEW

File: src/main/java/decide/LIC.java [hunk 1]
```diff
@@ -78,7 +109,7 @@ public boolean[] calculateCMV() {
         CMV[11] = LIC_11();
         CMV[12] = LIC_12();
         CMV[13] = LIC_13();
-        CMV[14] = LIC_14();
+        //CMV[14] = LIC_14();
```
Comment (tohanss) [inline src/main/java/decide/LIC.java:112]: Should not be commented out

**Review by tohanss — CHANGES_REQUESTED** [review]
Looks good! Just the commented out line

**Review by TerenceNg03 — CHANGES_REQUESTED** [review]
Please merge CI from main first, so the check will run on this pr.

File: src/main/java/decide/LIC.java (hunk 1 above)
Comment (TerenceNg03) [reply src/main/java/decide/LIC.java:112]: Can be fixed later.

**Review by tohanss — APPROVED** [review]

**Pull request merged**